analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContextBase.cs

```diff
@@ -123,6 +126,27 @@ public bool HasMatchingScope(DiagnosticDescriptor descriptor)
             descriptor.CustomTags.Contains(tag);
     }
 
+    private bool ShouldAnalyzeFile(SonarLintXmlReader reader, string filePath) =>
```

Comment:
Nitpick: IMO it brings more value if it is named `sonarLintXml` as later for me `sonarLintXml.TestInclusions` brings more value than `reader.TestInclusions`. WDYT?

```suggestion
    private bool ShouldAnalyzeFile(SonarLintXmlReader sonarLintXml, string filePath) =>
```